### DISCUSSION TITLE
[Question]: How to see multiple camera feeds of a specific moment in time?

### DISCUSSION THREAD

**Discussion by Rubeast** [blakeblackshear/frigate#10355] [Ask A Question]
### What is your question:

I record 24/7 and sometimes like to open the recordings at a specific time.
I know I can do that for 1 camera when I click the "recordings" button of a single camera.
But if, for example, a person walks from 1 camera to an other camera, you have to look for that specific moment again for a different camera?

I just would love to see a timeline on the "Cameras" screen where we have the overview of all cameras.
This seems like basic behaviour of a surveillance system so I'm wondering if I'm missing something? 
If not, how are other people doing this? Do I need to combine Frigate with an other system and do the 24/7 recording there?

**Comment by NickM-27** [comment]
This is currently not possible in 0.13 however we are rewriting the UI and the new UI will have this feature in multiple places.

**Comment by Rubeast** [comment]
Awesome! Thx!
